lib/factory_girl.rb

```diff
@@ -55,8 +57,16 @@ def self.reset_configuration
     @configuration = nil
   end
 
-  def self.lint(factories_to_lint = FactoryGirl.factories)
-    Linter.lint!(factories_to_lint)
+  def self.lint(factories = nil, options = {})
+    if factories.is_a?(Hash)
+      factories_to_lint = FactoryGirl.factories
+      options = factories
```

Comment:
this feels a bit strange to me - is there a way we could take in arguments and make this all more clear? I believe ActiveSupport has an `extract_options!` method that *might* be doing what you want